### ISSUE TITLE
per chat type listeners

### ISSUE THREAD

**Issue by hamidqp** [badfarm/zanzara#33]
I want listeners to recognize each chat type. I mean when i make new listener set that to just listen for updates from private chats 
or something like this: 
`$bot->onMessage(function (Context $ctx){},'private');`

so this function do not work when messages comes from group
(yes i can do it by 
` if ($ctx->getEffectiveChat()->getType() === 'private') {}`
but without that would great)

**Comment by cheeghi** [comment]
It sounds cool! We'll give it a look

**Comment by SQdev-git** [comment]
Please add the method $ bot-> onGroup (); this so that teams work only in a group or in private chats

**Comment by cheeghi** [comment]
Available in version `0.7.0`.

```php
<?php

$bot = new Zanzara("token");

$bot->onCommand('start', function (Context $ctx) {
    // ...
}, ['chat_type' => 'group']);

$bot->run();

```
